cmd/chisel/cmd_cut.go

```diff
@@ -92,11 +92,15 @@ func (cmd *cmdCut) Execute(args []string) error {
 			Components: archiveInfo.Components,
 			CacheDir:   cache.DefaultDir("chisel"),
 			Priority:   archiveInfo.Priority,
+			Pro:        archiveInfo.Pro,
 		})
 		if err != nil {
-			return err
+			if err != archive.ErrNoArchiveCredentials {
+				return err
+			}
```

Comment:
info: Could it happen that none of the archives opened and returned with `archive.ErrNoArchiveCredentials`? If so, should we return an error immediately or let the execution go to `slicer.Run()` below?

Reply:
Good question. If we continue to `slicer.Run()` we will print `slice package foo missing from archive` error. Maybe we should check that archives aren't empty in `cmd_cut.go` and print error about it otherwise.

Reply:
Added
```
       if len(archives) == 0 {
               return fmt.Errorf("no valid archives (%d skipped)", len(release.Archives))
       }
```